src/NATS.Client.Core/NatsConnectionPool.cs

```diff
@@ -52,11 +56,6 @@ public IEnumerable<NatsConnection> GetConnections()
         }
     }
 
-    public INatsConnection GetCommand()
```

Comment:
I removed this, because it's no identical to `INatsConnectionPool.GetConnection()`.